demo/demo.jl

```diff
@@ -0,0 +1,33 @@
+using Base.Test
+using TestSetExtensions
+using ReferenceTests
+
+
+@testset  ExtendedTestSet "all the tests" begin
```

Comment:
i am guessing this is just a leftover?

Reply:
No?
Run the demo, and it shows that the `ExtendedTestSet` works, and does it's core functionality of displaying Diffs.

Reply:
oh, I see. it's from `TestSetExtensions`. So to be clear, in case one doesn't care about the diff display, the functionality here itself doesn't require using `ExtendedTestSet` right?

Reply:
exactly.